test/bdd/steps/api/datasets/requests.json

```diff
@@ -1,73 +1,73 @@
-    "id": "did:dkg:hardhat/0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07/0",
-    "state": "0x759c285786b95622dad67a6be857a4eec3d9ba0caec991ed4297629ae6abbc0d"
-  },
-  "blockchainNotDefinedRequestBody": {
-    "publishType": "asset",
-    "assertionId": "0xc311cca6412f8453067ac7a04831af411b2963734d107541763c1ef7c8e56f65",
-    "assertion": [
-      "_:c14n0 <http://schema.org/born> \"Born: April 30, 1916, Petoskey, Michigan, United States\" .",
-      "_:c14n0 <http://schema.org/description> \"Claude Elwood Shannon was an American mathematician, electrical engineer, and cryptographer known as the father of information theory. \" .",
-      "_:c14n0 <http://schema.org/died> \"Died: February 24, 2001, Medford, Massachusetts, United States\" .",
-      "_:c14n0 <http://schema.org/name> \"Claude Shànnon\" .",
-      "_:c14n0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> ."
-    ],
-    "blockchain": null,
-    "contract": "0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07",
-    "tokenId": 0
-  },
-  "nonExistentUAL": {
-    "id": "did:dkg:hardhat/0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07/1"
-  },
-  "invalidUAL": {
-    "id": "did:dkg:hardhat/0xB0D4afd8879eD9F/52b28595d31B441D079B2Ca07/1"
-  },
-  "nonExistentState": {
-    "id": "did:dkg:hardhat/0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07/0",
-    "state": -1
-  },
-  "invalidStateHash": {
-    "id": "did:dkg:hardhat/0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07/0",
-    "state": "0x591503a1c8ba4667dd7afd203025c1bf594d817d8eec71274fe960d69fb8584e"
-  },
-  "validUpdateRequestBody": {
-    "assertionId": "0x591503a1c8ba4667dd7afd203025c1bf594d817d8eec71274fe960d69fb8584f",
-    "assertion": [
-      "<uuid:2> <http://schema.org/city> <uuid:Nis> .",
-      "<uuid:2> <http://schema.org/company> \"TL\" .",
-      "<uuid:2> <http://schema.org/user> <uuid:user:2> ."
-    ],
-    "blockchain": "hardhat",
-    "contract": "0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07",
-    "tokenId": 0
-  }
+    "validPublishRequestBody": {
+        "assertionId": "0xc311cca6412f8453067ac7a04831af411b2963734d107541763c1ef7c8e56f65",
+        "assertion": [
+            "_:c14n0 <http://schema.org/born> \"Born: April 30, 1916, Petoskey, Michigan, United States\" .",
+            "_:c14n0 <http://schema.org/description> \"Claude Elwood Shannon was an American mathematician, electrical engineer, and cryptographer known as the father of information theory. \" .",
+            "_:c14n0 <http://schema.org/died> \"Died: February 24, 2001, Medford, Massachusetts, United States\" .",
+            "_:c14n0 <http://schema.org/name> \"Claude Shànnon\" .",
+            "_:c14n0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> ."
+        ],
+        "blockchain": "test2",
+        "contract": "0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07",
+        "tokenId": 0
+    },
+    "validGetFirstStateRequestBody": {
+        "id": "did:dkg:blockchain/0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07/0",
```

Comment:
What about chain id here?

Reply:
It should work because of backwards compatibility, but can change